proxy/envoy/config.go

```diff
@@ -228,10 +288,15 @@ func buildInboundFilters(instances []*model.ServiceInstance) HTTPRouteConfigs {
 
 			http := httpConfigs.EnsurePort(instance.Endpoint.Port)
 			http.VirtualHosts = append(http.VirtualHosts, host)
+		case model.ProtocolTCP:
+			cluster := buildInboundCluster(instance.Endpoint.Port, port.Protocol)
+			route := buildTCPRoute(cluster, instance.Endpoint.Address, instance.Endpoint.Port)
```

Comment:
My understanding is that build{inbound,outbound}Cluster is common for all protocols. Were you suggesting otherwise?